src/test/kotlin/com/music/awesomemusic/controllers/AccountControllerTest.kt

```diff
@@ -42,64 +46,287 @@ class AccountControllerTest {
 
     lateinit var account: AwesomeAccount
 
-    @Before
-    fun prepareAccount() {
-        account = accountService.createAccount(AccountSignUpForm("holker", "123456", "test.mail@gmail.com", false))
-    }
+    // @Before
+    //  fun prepareAccount() {
+    //     account = accountService.createAccount(AccountSignUpForm("holker", "123456", "test.mail@gmail.com", false))
+    //  }
 
     @After
     fun deleteAccount() {
         tokenRepository.deleteAll()
         accountRepository.deleteAll()
     }
 
+
+    @Test
+    @Throws(Exception::class)
+    fun shouldSignInWithUsernameExpected200() {
+
+        accountService.createAccount(AccountSignUpForm("test1", "1234", "emailTest1", false))
+        val headers = HttpHeaders()
+        headers.contentType = MediaType.APPLICATION_JSON
+        val request = HttpEntity<String>("{\"login\": \"test1\", \"password\": \"1234\"}", headers)
+        val responseEntity = restTemplate.postForEntity("http://localhost:$port/api/user/sign-in", request, String::class.java)
+
+        assertEquals(responseEntity.statusCode, HttpStatus.OK)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun shouldSignInWithEmailExpected200() {
+
+        accountService.createAccount(AccountSignUpForm("test2", "1234", "emailTest2", false))
+        val headers = HttpHeaders()
+        headers.contentType = MediaType.APPLICATION_JSON
+        val request = HttpEntity<String>("{\"login\": \"emailTest2\", \"password\": \"1234\"}", headers)
+        val responseEntity = restTemplate.postForEntity("http://localhost:$port/api/user/sign-in", request, String::class.java)
+
+        assertEquals(responseEntity.statusCode, HttpStatus.OK)
+    }
+
+
+    @Test
+    @Throws(Exception::class)
+    fun shouldSignUpExpected201() {
+
+        val headers = HttpHeaders()
+        headers.contentType = MediaType.APPLICATION_JSON
+        val request = HttpEntity<String>("{\"username\": \"test3\", \"password\": \"123HAha@%/+'!#$^?:.(){}[]~-_.\",\"email\":\"test3@mail.ru\",\"is_collective\": \"false\" }", headers)
+        val responseEntity = restTemplate.postForEntity("http://localhost:$port/api/user/registration", request, String::class.java)
+
+        assertEquals(responseEntity.statusCode, HttpStatus.CREATED)
+
+    }
+
+
+    @Test
+    @Throws(Exception::class)
+    fun shouldSignUpToReturnToken() {
```

Comment:
```suggestion
    fun shouldSignInToReturnToken() {
```
SignUp doesn't return token.